integration_tests/integration/barcode/createBarcodeAddressImage.spec.ts

```diff
@@ -33,7 +33,7 @@ context('Create Barcode Image', () => {
     reviewRecipientsPage
       .addAnotherRecipient()
       .submitWithUnknownPrisonNumber()
-      .submitWithValidValues('Arry Ardnut')
+      .submitWithValidValues(2, 'Arry Ardnut')
```

Comment:
Not related to this change, but this was causing an IDE error (and presumably a runtime test error??) because the first arg to the function is now the contact ID not the name